packages/nitro-utils/src/nitro/patchEventHandler.ts

```diff
@@ -0,0 +1,39 @@
+import { getDefaultIsolationScope, getIsolationScope, logger, withIsolationScope } from '@sentry/core';
+import type { EventHandler } from 'h3';
+import { flushIfServerless } from '../util/flush';
+
+/**
+ * A helper to patch a given `h3` event handler, ensuring that
+ * requests are properly isolated and data is flushed to Sentry.
+ */
+export function patchEventHandler(handler: EventHandler): EventHandler {
```

Comment:
L: We could give this a name that speaks more for itself (with the downside that it gets long). E.g. forkScopeAndFlushServerless

Reply:
I opted for a shorter name because where it's imported from gives enough context. But happy to change if you think otherwise, let me know :)

Reply:
No strong feelings about this :D

Reply:
Will keep it then, can always change later since this isn't really public api.